R/estimate_KM.R

```diff
@@ -157,7 +175,8 @@ vr_KM_est <- function(
       attr(survfit_object$strata, "names") <- as.character(data[1, main])
     }
   }
-  
-  #### Return ####
+
+# Return ------------------------------------------------------------------
+
   return(survfit_object)
```

Comment:
I get the following error:

Call: estimate_KM(adtte, strata = "TRTP")
Message: Error in Surv(AVAL, 1 - CNSR) : could not find function "Surv"